core/src/Docs/Docs.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBook } from "@fortawesome/free-solid-svg-icons";
+import { Button } from "../Button"
+
+export function Docs(props) {
+
+  function openDocs() {
+    let url=`https://zesty.org/?q=${props.subject}`
+    if(props.url){
+      url = props.url
+    }
+    window.open(url, '_blank');
+  }
+
+  return (
+    <Button title={`Open documentation for ${props.subject}`} size="small" kind="documentation" onClick={openDocs}>
```

Comment:
This would be more semantic as a link, i.e. `<a href></a>`. Is there a reason we need to use a button and onClick handler?